test/benchmarks/bench_classify.rb

```diff
@@ -51,7 +51,7 @@ def bench_allocations_with_cache
     Primer::Classify::Cache.preload!
     Primer::Classify.call(**@values)
 
-    assert_allocations 35 do
+    assert_allocations 30 do
```

Comment:
😱 was not expecting that

Reply:
I think this will be one of the benefits of auto-generating a lot of our API. Since the values are already determined in the YAML file, it's implicitly caching them.